src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -64,23 +64,41 @@ image_pyramid:
     contains-pii: false
     vitessce-hints: ['is_image', 'pyramid']
     
-IMC:
-    description: Imaging Mass Cytometry
-    alt-names: []
+IMC2D:
+    description: 2D Imaging Mass Cytometry
+    alt-names: ['IMC', '2D-IMC']
     primary: true
     contains-pii: false
     vitessce-hints: []
 
-3D-IMC:
+IMC3D:
     description: 3D Imaging Mass Cytometry
-    alt-names: []
+    alt-names: ['3D-IMC']
```

Comment:
```suggestion
    alt-names: ['IMC-3D']
```

Reply:
We can't follow the same policy with alt-names because they are intended to capture and correct existing database entries, replacing the old value '3D-IMC' with the new value IMC3D (with the description we agree on).  3D-IMC is the value that must be replaced so it needs to be the alt-name.

Reply:
Ok, then let's just do it for those where the change can be made, i.e. not `alt-names`.